server/models/migrations/next/migrate.sql

```diff
@@ -79,3 +79,15 @@ ALTER TABLE `service` DROP COLUMN `id`;
 
 
 ALTER TABLE `service` ADD PRIMARY KEY (`uuid`);
+  ('collectionCapacity', 'REPORT.COLLECTION_CAPACITY.TITLE');
```

Comment:
I discovered what the issue was.  This migration script doesn't compile.

Can you remove the `collectionCapacity` part?  It breaks the script.

Reply:
Okay